app/assets/javascripts/initializers/initializeCommentDropdown.js

```diff
@@ -126,7 +138,13 @@ function initializeCommentDropdown() {
     dropdown.addEventListener('click', dropdownFunction);
   }
 
+  function copyPermalinkButtonListener(copyPermalinkButton) {
```

Comment:
Refactor (non-blocking): adding `ButtonListener` is redundant

Reply:
@nickytonline would you prefer I changed it to `copyPermalinkListener` only?

```suggestion
  function copyPermalinkListener(copyPermalinkButton) {
```

Reply:
No that's fun and thanks again!